xml/dm_registers.xml

```diff
@@ -424,30 +476,46 @@
 
             This field only contains a valid value if \FdmAbstractcsBusy is 0.
 
-            0 (none): No error.
+            <value v="0 " name="none">
```

Comment:
All of the values in this field have spaces in the `v=` string.

Reply:
But you should've seen the macro I created to automatically make the XML tags and put the value in the right place!